### PR TITLE
VIITE-2479_minor_discontinuity_change_to_Major

### PR DESCRIPTION
**Pull request by peltsi212**

### CODE REVIEW

**Comment by peltsi212** [comment]
Tests are ok in jenkins

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ class ProjectValidator {
       *
       * @return
       */
-    def checkDiscontinuityOnLastPart: Seq[ValidationErrorDetails] = {
+    def checkDiscontinuityOnLastLinkPart() = {
```
Comment (oskarmalmstrom) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala:1066]: Has the return type changed or just removed?

Comment (peltsi212) [reply digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala:1066]: I do not remember that I have changed this. Could it be that Idea has removed it with automatic formatting? I think that return type is now assigned from type of returning function.

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala [hunk 2]
```diff
@@ -1979,7 +1979,7 @@ Left|      |Right
     }
   }
 
-  test("Test checkRoadContinuityCodes When there is Minor discontinuous ending in ramp road between parts (of any kind) Then should not give any error") {
+  test("Test checkRoadContinuityCodes When there is Major discontinuous ending in ramp road between parts (of any kind) Then should not give any error") {
```
Comment (skosonen) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala:1982]: Remove word "Major".

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala [hunk 3]
```diff
@@ -2026,11 +2026,11 @@ Left|      |Right
       val updProject = projectService.fetchProjectById(project.id).get
       mockEmptyRoadAddressServiceCalls()
       projectValidator.checkRoadContinuityCodes(updProject,
-        starting ++ last.map(_.copy(discontinuity = Discontinuity.MinorDiscontinuity)), isRampValidation = true).distinct should have size 0
+        starting ++ last.map(_.copy(discontinuity = Discontinuity.Discontinuous)), isRampValidation = true).distinct should have size 0
     }
   }
 
-  test("Test checkRoadContinuityCodes When next part exists in road address / project link table and is not connected Then Project Links could be both Minor discontinuity or Discontinuous") {
+  test("Test checkRoadContinuityCodes When next part exists in road address / project link table and is not connected Then Project Links could be both only Major Discontinuous") {
```
Comment (skosonen) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala:2033]: Remove "Major".

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala [hunk 4]
```diff
@@ -2427,6 +2427,55 @@ Left|      |Right
     }
   }
 
+  test("Test checkRoadContinuityCodes for continuous, MinorDiscontinuity and MajorDiscontinuity") {
```
Comment (skosonen) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala:2430]: MajorDiscontinuity -> Discontinuous

**Pull request merged**